apps/hubble/src/cli.ts

```diff
@@ -71,6 +71,10 @@ app
   .option("-c, --config <filepath>", "Path to the config file.")
   .option("--db-name <name>", "The name of the RocksDB instance. (default: rocks.hub._default)")
   .option("--process-file-prefix <prefix>", 'Prefix for file to which hub process number is written. (default: "")')
+  .option(
+    "--log-individual-messages",
```

Comment:
nit: this is kind of a long flag and unclear which messages it's logging, though don't feel strongly

```suggestion
    "--log-submit-messages",
```